src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -24,7 +24,8 @@ public abstract class LbfgsTrainerBase<TOptions, TTransformer, TModel> : Trainer
       where TOptions : LbfgsTrainerBase<TOptions, TTransformer, TModel>.OptionsBase, new()
     {
         /// <summary>
-        /// Base options for <a href='https://en.wikipedia.org/wiki/Limited-memory_BFGS'>L-BFGS</a>-based trainers.
+        /// Options for the <see cref="LbfgsTrainerBase{TOptions, TTransformer, TModel}"/>.
+        /// It is used in <a href='https://en.wikipedia.org/wiki/Limited-memory_BFGS'>L-BFGS</a>-based trainers.
```

Comment:
Would it be better to have this wikipedia link on the trainer itself?

Reply:
I kind of feel adding link to this abstract class could make users thinking they can create it.